webgui/package.json

```diff
@@ -14,7 +14,7 @@
     "codemirror"       : ">=5.19.0",
     "jquery.fancytree" : ">=2.19.0",
     "svg-pan-zoom"     : ">=2.19.0",
-    "marked"           : ">=0.3.6",
+    "marked"           : "2.1.3",
```

Comment:
This should be a minimal version, not an exact one.

Reply:
Especially semantic versioning should be used.

> **Tilde Version Range (~)**
The ~ operator is best explained by example: ~1.2 is equivalent to >=1.2 <2.0.0, while ~1.2.3 is equivalent to >=1.2.3 <1.3.0. As you can see it is mostly useful for projects respecting [semantic versioning](https://semver.org/). A common usage would be to mark the minimum minor version you depend on, like ~1.2 (which allows anything up to, but not including, 2.0). Since in theory there should be no backwards compatibility breaks until 2.0, that works well. Another way of looking at it is that using ~ specifies a minimum version, but allows the last digit specified to go up.
> 
> **Caret Version Range (^)**
The ^ operator behaves very similarly, but it sticks closer to semantic versioning, and will always allow non-breaking updates. For example ^1.2.3 is equivalent to >=1.2.3 <2.0.0 as none of the releases until 2.0 should break backwards compatibility. For pre-1.0 versions it also acts with safety in mind and treats ^0.3 as >=0.3.0 <0.4.0 and ^0.0.3 as >=0.0.3 <0.0.4.
> This is the recommended operator for maximum interoperability when writing library code.